gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -65,13 +81,21 @@ public Bundle mapToBundle(RCMRIN030000UK06Message xmlMessage) {
         var referralRequests = mapReferralRequests(ehrFolder, patient);
         addEntries(bundle, referralRequests);
 
-        var observations = mapObservations(ehrExtract, patient, List.of()); //TODO: Provide list of encounters
+        var observations = mapObservations(ehrExtract, patient, encounters);
         addEntries(bundle, observations);
 
+        // TODO: Add references to mapped resources in their appropriate lists
+        addEntries(bundle, topics);
```

Comment:
Is there any reason these can't be done above with the others?

Reply:
I've put this at the bottom because we will need some functionality after mapping all the other resources to then add them to the appropriate topic and category lists in my next ticket. This might change but im not sure. For now it's the most sensible spot for it.